nbpackage/src/main/resources/org/apache/netbeans/nbpackage/rpm/rpm.spec.template

```diff
@@ -0,0 +1,23 @@
+Name:           ${RPM_PACKAGE}
+Version:        ${RPM_VERSION}
+Release:        0
+Summary:        ${RPM_SUMMARY}
+License:        Apache License 2.0
+Group:          Development/Tools/IDE
+URL:            https://netbeans.apache.org/
+BuildArch:      ${RPM_ARCH}
+Packager:       ${RPM_MAINTAINER}
+Vendor:         Apache NetBeans
+Requires:       java >= 11
```

Comment:
Recommends instead of Requires as the user could already have a JDK (zip) instead of having the corresponding rpm installed?

Reply:
OK, so that line should be written on demand (JDK vs no JDK shipped).

Reply:
I assume Recommends works similarly to DEB, in that packaging tools will usually prompt to install, but it can be installed without error if someone did want to use a JDK they'd installed in some other fashion?

If the runtime is bundled, then there is no need for the JDK at all.  That's why it's only added in the DEB when no JDK is included.

Reply:
Understood. I have removed "Requires" and left "Recommends" (weak dependency); ` java-devel >= 11`